### PR TITLE
[CHEC-977] Update tag styling

### PR DESCRIPTION
**Pull request by kvisca**
Screen Shot 2020-10-23 at 12 22 44 PM

### CODE REVIEW

File: src/components/ChecTag.vue [hunk 1]
```diff
@@ -110,6 +110,14 @@ export default {
     .chec-tag__dismiss {
       @apply bg-white text-gray-600;
     }
+
+    &.chec-tag--disabled {
+      @apply bg-gray-600 text-white cursor-not-allowed border-none;
```
Comment (robbieaverill) [inline src/components/ChecTag.vue:115]: Can this line go up into line 92, and the line below up to line 95?

Comment (ScopeyNZ) [inline src/components/ChecTag.vue:115]:
```suggestion
      @apply bg-gray-600 text-white;
```

These styles have a specificity of 2, so lets keep the number of styles we're defining nice and low.

File: src/components/ChecTag.vue [hunk 2]
```diff
@@ -110,6 +110,14 @@ export default {
...
+
+      .chec-tag__dismiss {
+        @apply cursor-not-allowed text-gray-600;
```
Comment (ScopeyNZ) [inline src/components/ChecTag.vue:118]:
```suggestion
        @apply text-gray-600;
```

**Comment by robbieaverill** [comment]
ping @kvisca

**Comment by kvisca** [comment]
@robbieaverill Thanks for the ping, updated. Let me know if that change works 👍

File: src/stories/components/ChecTag.stories.mdx [hunk 3]
```diff
@@ -27,6 +27,9 @@ import ChecTagGroup from '../../components/ChecTagGroup';
           type: Boolean,
           default: () => boolean('Active', false),
         },
+        background: {
+          default: select('Background Color', ['light', 'dark']),
```
Comment (robbieaverill) [inline src/stories/components/ChecTag.stories.mdx:31]:
```suggestion
          default: select('Background color', ['light', 'dark']),
```

**Review by robbieaverill — APPROVED** [review]

**Pull request merged**